test/unit/rubygem_searchable_test.rb

```diff
@@ -224,30 +224,35 @@ class RubygemSearchableTest < ActiveSupport::TestCase
       context "Elasticsearch::Transport::Transport::Errors::BadRequest" do
         setup do
           @ill_formated_query = "updated:[2016-08-10 TO }"
-          Rubygem.stubs(:legacy_search).returns Rubygem.all
-          @error_msg, = Rubygem.search(@ill_formated_query, elasticsearch: true)
         end
 
         should "fallback to legacy search" do
-          assert_received(Rubygem, :legacy_search) { |arg| arg.with(@ill_formated_query) }
+          Rubygem.expects(:legacy_search).with(@ill_formated_query).returns(Rubygem.all)
+
+          Rubygem.search(@ill_formated_query, elasticsearch: true)
         end
 
         should "give correct error message" do
           expected_msg = "Failed to parse: '#{@ill_formated_query}'. Falling back to legacy search."
+
+          @error_msg, _ = Rubygem.search(@ill_formated_query, elasticsearch: true)
```

Comment:
test/unit/rubygem_searchable_test.rb:238:23: C: Style/TrailingUnderscoreVariable: Do not use trailing _s in parallel assignment. Prefer @error_msg, = Rubygem.search(@ill_formated_query, elasticsearch: true)